mets_retriever/mets_retriever.py

```diff
@@ -99,14 +128,18 @@ def _filter_aips(self, aips):
             if aip["status"] == "UPLOADED" and aip["replicated_package"] is None
         ]
 
-    def download_all_mets_files(self):
+    def download_all_mets_files(self, with_replicas_only=False):
         am = AMClient(
             ss_url=self.storage_service_url,
             ss_user_name=self.storage_service_username,
             ss_api_key=self.storage_service_api_key,
         )
         aips = self._filter_aips(am.aips())
         for aip in aips:
+            if with_replicas_only:
+                if not aip.get("replicas"):
```

Comment:
I think it's better to combine these two "if" statements into a single "if" with an "and" operator.